### DISCUSSION TITLE
Could maybe update someone the dockerhub Overview

### DISCUSSION THREAD

**Discussion by Tschak77** [LMS-Community/slimserver-platforms#47] [General]
Hi could maybe someone update the dockerhub Overview with the environment variable

-e TZ=Europe/Berlin (as example)

this should work because you have the package tzdata installed.

For example for truenas it is not possible to set volume outsite of /mnt . For theses system it could be a solution and many other Container do it in same way.

**Comment by michaelherger** [comment]
Did you try it? Does it work? If it's only a question about documenting the variable, then I can easily do this.

**Comment by Tschak77** [comment]
yes it is working and a common way

https://tecadmin.net/change-timezone-in-docker-container/
https://dev.to/0xbf/set-timezone-in-your-docker-image-d22

```
root@lms01-ix-chart-859f578974-ntgxc:/config# date
Mon Nov  6 23:27:42 CET 2023
```

**Reply by michaelherger** [reply]
Thanks! I'll add a line or two to the readme file.

**Reply by Tschak77** [reply]
Thanks very much
